.travis.yml

```diff
@@ -40,3 +40,4 @@ deploy:
   distributions: sdist
   on:
     tags: true
+    jdk: openjdk8
```

Comment:
Ah, nice. Didn't realize it was that simple. I've been moving to the `jobs` format elsewhere to handle this. :+1: